airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/AirbyteExceptionHandler.java

```diff
@@ -23,7 +24,7 @@ public void uncaughtException(Thread t, Throwable e) {
     // the sync."
     // from the spec:
     // https://docs.google.com/document/d/1ctrj3Yh_GjtQ93aND-WH3ocqGxsmxyC3jfiarrF6NY0/edit#
-    LOGGER.error(logMessage, e);
+    LOGGER.error(logMessage + ExceptionUtils.getStackTrace(e));
```

Comment:
is this still needed?

Reply:
I hope this is not needed anymore. I think @yurii-bidiuk have not pushed the new changes to this branch yet, because @evantahler 's changes are not merged yet

Reply:
@grishick yes, you're right, thanks. I'm waiting until @evantahler 's changes will be merged and then will update this PR.